perl/LICENSE.txt

```diff
@@ -0,0 +1,24 @@
+The MIT License (MIT)
```

Comment:
There should only be one master LICENSE file, which is copied to all implementations with Make. Feel free to add your name to the parent LICENSE file.